docs/concepts/pre-mid-post.md

```diff
@@ -0,0 +1,31 @@
+
+# PRE/MID/POST scripts
+The scripts are located in` /etc/perun/[service_name].d/`
+
+The expected filename is `[pre/mid/post]_[number]_filename.`
+
+There can be multiple scripts, they are run in alphabetical order, the `[number]` part, which typically starts with 10, is
+used to determine priority.
+
+#### Exmaples: `pre_10_set_variables.sh`, `mid_20_change_behavior.sh`, `post_30_housekeeping.sh`
```

Comment:
If I understand it correctly, the order is always 1. pre, 2. mid and 3. post ... and the number is just for order scripts of the given type - `pre_10`, `pre_20`. This way the example is presented it could look like the number is the main order mechanism, but it should be (at least I think) just the second rule for ordering.

Reply:
Yes, you are right. I did not understand it correctly at first, but you are right.